lib/client-assets.php

```diff
@@ -55,9 +55,10 @@ function wc_admin_register_script() {
 	$content     = 'wp.i18n.setLocaleData( ' . json_encode( $locale_data ) . ', "wc-admin" );';
 	wp_add_inline_script( 'wc-components', $content, 'before' );
 
-	// Add Tracks script to the DOM if tracking is opted in.
+	// Add Tracks script to the DOM if tracking is opted in, and Jetpack is installed/activated.
 	$tracking_enabled = 'yes' === get_option( 'woocommerce_allow_tracking', 'no' );
-	if ( $tracking_enabled ) {
+	$jetpack_active = is_plugin_active( 'jetpack/jetpack.php' );
```

Comment:
Maybe check for `JETPACK__VERSION` in case the plugin is installed in a different folder?